[2024-10-04T22:53:43.741+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: create_table_in_postgres.create_table_task manual__2024-10-04T22:53:41.541035+00:00 [queued]>
[2024-10-04T22:53:43.751+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: create_table_in_postgres.create_table_task manual__2024-10-04T22:53:41.541035+00:00 [queued]>
[2024-10-04T22:53:43.752+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-10-04T22:53:43.770+0000] {taskinstance.py:2191} INFO - Executing <Task(PostgresOperator): create_table_task> on 2024-10-04 22:53:41.541035+00:00
[2024-10-04T22:53:43.779+0000] {standard_task_runner.py:60} INFO - Started process 281 to run task
[2024-10-04T22:53:43.782+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'create_table_in_postgres', 'create_table_task', 'manual__2024-10-04T22:53:41.541035+00:00', '--job-id', '124', '--raw', '--subdir', 'DAGS_FOLDER/create_mysql_table.py', '--cfg-path', '/tmp/tmpwblo7y7k']
[2024-10-04T22:53:43.786+0000] {standard_task_runner.py:88} INFO - Job 124: Subtask create_table_task
[2024-10-04T22:53:43.844+0000] {task_command.py:423} INFO - Running <TaskInstance: create_table_in_postgres.create_table_task manual__2024-10-04T22:53:41.541035+00:00 [running]> on host 005b6f55f237
[2024-10-04T22:53:43.952+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='create_table_in_postgres' AIRFLOW_CTX_TASK_ID='create_table_task' AIRFLOW_CTX_EXECUTION_DATE='2024-10-04T22:53:41.541035+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-04T22:53:41.541035+00:00'
[2024-10-04T22:53:43.956+0000] {sql.py:276} INFO - Executing: 
        CREATE TABLE IF NOT EXISTS clients (
            id SERIAL PRIMARY KEY,
            nom VARCHAR(100) NOT NULL,
            email VARCHAR(100) UNIQUE NOT NULL
        );
        
[2024-10-04T22:53:43.975+0000] {base.py:83} INFO - Using connection ID 'postgres_conn_id' for task execution.
[2024-10-04T22:53:43.989+0000] {base.py:83} INFO - Using connection ID 'postgres_conn_id' for task execution.
[2024-10-04T22:53:44.152+0000] {sql.py:450} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS clients (
            id SERIAL PRIMARY KEY,
            nom VARCHAR(100) NOT NULL,
            email VARCHAR(100) UNIQUE NOT NULL
        );
        , parameters: None
[2024-10-04T22:53:44.175+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=create_table_in_postgres, task_id=create_table_task, execution_date=20241004T225341, start_date=20241004T225343, end_date=20241004T225344
[2024-10-04T22:53:44.238+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-10-04T22:53:44.259+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
